core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -585,21 +601,20 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
           x: moment(dataPoint.Timestamp).valueOf(),
           y: parseInt(dataPoint[valueProperty])
         };
-
-        if (!obj['y']) {
-          obj['y'] = parseInt(dataPoint[valueProperty.toLowerCase()])
-        }
         return obj;
       });
 
     let timeRange = this.filters.getFieldValueOfLabel('TIME RANGE');
+    this.aggregation = this.filters.getFieldValueOfLabel('AGGREGATION') == 'sum' ? 'Sum': 'Avg.';
```

Comment:
Are we assuming Sum and Average are the only valid values? Shouldn't it be more generic or have a mapping instead?

Reply:
Since in dropdown of aggregation, there are two values - sum and avg, so there are the only valid ones as per my understanding.

Reply:
@dmpl1 Once we bring in Azure, we have some others as well. Might be better to move this to a mapping.

Reply:
Lets defer this to once Azure changes comes in.